### PR TITLE
test: complement unittest for `Validation` command

### PR DESCRIPTION
**Pull request by moreal**
SSIA.

### CODE REVIEW

File: __tests__/main/headless/validation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    });
+
+    context("with valid private key", function () {
+      const validPrivateKeys = [
+        "ed5eba446d4c7cb50d7d5eb72c732773d89ee336f1880ee237b07589ed2387fc",
+        "243389a88fcb6956aa3fb23acd3b770fb05bcdb993b326c19b1e40918f03155a",
+        "11",
+      ];
+      validPrivateKeys.forEach((validPrivateKey) => {
+        it(`It returns false with "${validPrivateKey}"`, function () {
```
Comment (boscohyun) [inline __tests__/main/headless/validation.spec.ts:29]:
```suggestion
        it(`It returns true with "${validPrivateKey}"`, function () {
```

File: __tests__/constant.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { join } from "path";
+
+export const HEADLESS_PATH = join(
```
[the remark this answers is not in the document]
Comment (moreal) [reply __tests__/constant.ts:3]: It seems *yes*. https://github.com/planetarium/9c-launcher/runs/2233163109?check_suite_focus=true#step:11:346

**Review by riemannulus — APPROVED** [review]

**Comment by moreal** [comment]
@boscohyun Could you review this pull request again please? 🙏🏻

**Comment by boscohyun** [comment]
> @boscohyun Could you review this pull request again please? 🙏🏻

@moreal https://github.com/planetarium/9c-launcher/pull/617#discussion_r604539687 was not resolved.

**Comment by moreal** [comment]
> @moreal [#617 (comment)](https://github.com/planetarium/9c-launcher/pull/617#discussion_r604539687) was not resolved.

I had a mistake to miss the commit to apply. 😢 I applied it again in 88196f2.

**Pull request merged**